README.md

```diff
@@ -14,6 +14,14 @@ First introducing core concepts such as variables, loops, conditionals and lists
 
 ### 3) Homology Modelling
 
+Contains a practical tutorial and lecture slides that aim to:
+
```

Comment:
White space is probably not necessary here? (not sure how it looks in Markdown).